web/src/components/Transform.vue

```diff
@@ -27,6 +29,13 @@ export default {
       normalize_methods,
       transform_methods,
       scaling_methods,
+      visiblePlots: {
+        score: true,
+        scree: true,
+        loadings: true,
+      },
+      cellSize: 300,
+      plotHeader: 48 + 5,
```

Comment:
Could you add a comment about where 48 and 5 came from?  +1 for separating them out an not just writing `53`.

Reply:
can be removed when https://github.com/girder/viime/issues/256 is implemented